app/javascript/elm/src/Main.elm

```diff
@@ -988,38 +987,34 @@ viewCrowdMapOptions isCrowdMapOn crowdMapResolution selectedSession tooltipIcon
         ]
 
 
-viewCrowdMapCheckBox : Bool -> Path -> Html Msg
-viewCrowdMapCheckBox isCrowdMapOn tooltipIcon =
+viewCrowdMapToggle : Bool -> Path -> Html Msg
+viewCrowdMapToggle isCrowdMapOn tooltipIcon =
     div [ class "filters__toggle-group" ]
-        [ input
-            [ id "checkbox-crowd-map"
-            , type_ "checkbox"
-            , checked isCrowdMapOn
-            , Events.onClick ToggleCrowdMap
-            ]
-            []
-        , label [ for "checkbox-crowd-map" ] [ text "Crowd Map" ]
+        [ label [] [ text "CrowdMap:" ]
```

Comment:
All the other labels are lower case. Is this an own name (nazwa własna)? If not I guess it should be consistent with the rest.

Reply:
this was explicitly requested by Michael